app/views/wikis/_form.html.erb

```diff
@@ -17,11 +17,13 @@
      <%= f.label :body %>
      <%= f.text_area :body, rows: 8, class: 'form-control', placeholder: "Enter wiki body" %>
    <% end %>
-    <div class="form-group">
-         <%= f.label :private, class: 'checkbox' do %>
-           <%= f.check_box :private %> Private wiki
-         <% end %>
-       </div>
+   <% if current_user.present? && (current_user.admin? || current_user.premium?) %>
```

Comment:
Well, those aren't actually Pundit conditionals. But what's the correct way to do it? Make a method in WikisController?